packages/dds/tree/src/feature-libraries/defaultFieldKinds.ts

```diff
@@ -532,29 +605,63 @@ const optionalChangeRebaser: FieldChangeRebaser<OptionalChangeset> = isolatedFie
 			if (over.fieldChange !== undefined) {
 				const wasEmpty = over.fieldChange.newContent === undefined;
 
-				// We don't have to rebase the child changes, since the other child changes don't apply to the same node
+				// TODO: Handle rebasing child changes.
```

Comment:
Nit: may as well be specific :)
```suggestion
				// TODO: Handle rebasing child changes over `over.childChange`
```

Reply:
Committed as suggested 😄